spec/javascripts/NewService/NewServiceForm.spec.jsx

```diff
@@ -41,6 +43,11 @@ it('should render the correct form depending on which mode is selected', () => {
   expect(wrapper.find(ServiceDiscoveryForm).exists()).toEqual(false)
 })
 
+it('should not render the ckeckbox to mark if it will act as product', () => {
+  const wrapper = mount(<NewServiceForm {...props}/>)
+  expect(wrapper.find('input#act_as_product').exists()).toEqual(false)
```

Comment:
You correct me if I'm wrong but I think the relevant attribute for Rails is `name="service[act_as_product]"`. If so, better assert that one:
```suggestion
  expect(wrapper.find('input[name="service[act_as_product]"]').exists()).toEqual(false)
```

EDIT
In fact, I think the proper way is to assert here de component `SelectBackendApi`, since the responsibility of `NewServiceForm` is to render de component nothing else.
Then, in **SelectBackendApi.spec.jsx** we make all the specific assertions like the input is rendered and the select works, etc.